digdag-core/src/main/java/io/digdag/core/log/LocalFileLogServerFactory.java

```diff
@@ -177,12 +180,17 @@ public LocalFileDirectTaskLogger(LogFilePrefix prefix, String taskName)
                 Path dir = getPrefixDir(dateDir, attemptDir);
                 Files.createDirectories(dir);
 
+                String fileName = LogFiles.formatFileName(taskName, Instant.now(), agentId.toString());
+                this.path = dir.resolve(fileName);
+
                 String temporaryFileName = LogFiles.formatPlainTextFileName(taskName, Instant.now(), agentId.toString());
                 this.temporaryPath = dir.resolve(temporaryFileName);
-                this.temporaryOutput = Files.newOutputStream(temporaryPath, CREATE, APPEND);
 
-                String conclusiveFileName = LogFiles.formatFileName(taskName, Instant.now(), agentId.toString());
-                this.conclusivePath = dir.resolve(conclusiveFileName);
+                if (realtimeFollow) {
+                    this.output = Files.newOutputStream(temporaryPath, CREATE, APPEND);
+                } else {
```

Comment:
Could you follow the code format which is applied to Digdag?

Reply:
OK, thanks.